packages/frontend/src/pages/insight-editor/insight-draft-switcher.tsx

```diff
@@ -225,16 +225,18 @@ export const InsightDraftSwitcher = ({ insight, onRefresh }) => {
                         align={{ base: 'stretch', md: 'center' }}
                         spacing={{ base: 0, md: '1rem' }}
                         ml="1rem"
-                        mb="0.25rem"
+                        mt={{ base: '0.5rem', md: '0.25rem' }}
+                        mb={{ base: '0.5rem', md: '0.25rem' }}
```

Comment:
You can combine these into `my`, short for margin Y-axis

Reply:
Oh I didn't know about that, I'll update it